C++/C++.sublime-syntax

```diff
@@ -582,6 +582,16 @@ contexts:
                 - include: global
             - include: expressions
 
+  test-invalid-module-partition:
+    - match: \s*(\:)
+      captures:
+        1: invalid.illegal.unexpected-character.c++
+      set:
+        - match: '.*$'
```

Comment:
1. A consuming pattern isn't required in that case as no other `match` directives are present.
2. We propably want to pop in front of a semicolon to keep it properly highlighted?

Maybe something like ...

```suggestion
        - match: $|(?=;)
```

Reply:
Good idea.  I forgot that other themes could highlight the semi-colon.